flow/skia_gpu_object.cc

```diff
@@ -39,6 +43,11 @@ void SkiaUnrefQueue::Drain() {
   for (SkRefCnt* skia_object : skia_objects) {
     skia_object->unref();
   }
+
+  if (context_ != nullptr) {
+    TRACE_EVENT0("flutter", "SkiaUnrefQueue::Drain performDeferredCleanup");
```

Comment:
Do we want to trace the duration of this?

Did you work out whatever issues were causing this to take 16ms+?

Reply:
I can remove that. I just figured out that our Drain task has already had a throttling of once every 3 seconds so a dozen milliseconds of cleanup is fine. The speed difference is due to the difference between the precompiled debug engine and my local debug engine (precompiled probably has opt while local one has unopt build). Switching to profile would eliminate the difference...

Reply:
I'm not concerned with removing it :) But this seems like a nice candidate for `TRACE_DURATION` rather than `TRACE_EVENT0`.